src/components/Molecules/NavbarLeft/index.jsx

```diff
@@ -8,16 +8,11 @@ const NavBarLeftSC = styled.div`
   align-items: center;
 `;
 
-const handleChange = (event) => {
-  // TODO
-  console.log(event);
-};
-
-function NavbarLeft() {
+function NavbarLeft(props) {
```

Comment:
You can destructure `onKeyDown` directly when passing component props:

```suggestion
function NavbarLeft({ onKeyDown }) {
```